config/i18n.json

```
@@ -55,6 +55,14 @@
     "Vault": "Vault",
     "Weapons": "Weapons"
   },
+  "BulkNote": {
+    "Title": "Change notes for 1 item",
+    "Title_plural": "Change notes for {{count}} items",
+    "Replace": "Replace notes",
+    "Append": "Append to notes / add tags",
+    "Remove": "Remove from notes / remove tags",
```

```suggestion
    "Append": "Append to notes / add #hashtags",
    "Remove": "Remove from notes / remove #hashtags",
```
'tags' might be misleading because we refer to `Keep` / `Junk` / `Infuse` etc. as tags elsewhere.